src/app/core/components/content-top-bar/content-top-bar.component.ts

```diff
@@ -28,6 +31,10 @@ export class ContentTopBarComponent {
     delay(0),
   );
 
+  groupInfo$ = this.currentContentService.content$.pipe(
```

Comment:
This file is generic and should have anything specific to groups. To be checked  after the previous remarks.

Reply:
You cannot just have a variable called `groupInfo$` is a generic component.. what does it mean when the content is an item?
Actually we do the same just above for items. Just use `currentContent.type === 'group'` to check that it is a group and access the detail info with `$any(...)`.

Reply:
So they will disappear with my previous remark.